iOS/ExposureNotification/source/ExposureNotification.csproj

```diff
@@ -5,8 +5,8 @@
     <IsBindingProject>true</IsBindingProject>
     <RootNamespace>ExposureNotification</RootNamespace>
     <AssemblyName>Xamarin.iOS.ExposureNotification</AssemblyName>
-    <AssemblyVersion>1.0.0.0</AssemblyVersion>
-    <FileVersion>1.0.0.0</FileVersion>
+    <AssemblyVersion>1.1.0.0</AssemblyVersion>
+    <FileVersion>1.1.0.0</FileVersion>
```

Comment:
Do we want to do this? Do we not keep the actual assembly version at 1.0.0.0 and just bump the information part?